internal/clusterinfo/types.go

```diff
@@ -77,11 +78,29 @@ func (p *Producer) Address() string {
 }
 
 func (p *Producer) HTTPAddress() string {
-	return fmt.Sprintf("%s:%d", p.BroadcastAddress, p.HTTPPort)
+	return addressWithPort(p.BroadcastAddress, p.HTTPPort)
```

Comment:
go has builtins for this, let's use [`net.JoinHostPort`](https://golang.org/pkg/net/#JoinHostPort)